guides/assets/javascripts/guides.js

```diff
@@ -52,11 +49,7 @@
     guidesIndexItem.value = currentGuidePath.substring(currentGuidePath.lastIndexOf("/") + 1) || 'index.html';
 
     guidesIndexItem.addEventListener("change", function(e) {
-      if (Turbolinks.supported) {
-        Turbolinks.visit(e.target.value);
-      } else {
-        window.location = e.target.value;
-      }
+      Turbo.visit(e.target.value);
```

Comment:
Turbo itself checks if a link is navigable and falls back to `window.location` if it's not, see:
https://github.com/hotwired/turbo/blob/aeeaae8edb5f6ec6ef6b19eaf93dc060a1e67b10/src/core/drive/navigator.ts#L28-L31